### DISCUSSION TITLE
Migration installer error on pimcore 10.6v

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [pimcore/pimcore#15711] [Q&A / Support] [answered]
### Pimcore version

pimcore 10.6

### Steps to reproduce

HI Team, 
in my existing bundle, im getting this error in pimcore 10.6 version migration installer error, 

### Actual Behavior

this installer not able find out it in vendor as its using in earlier version, in the current version its not using, so can anyone know which will be the alternative method to use instead of this?

pimcore demo its not used,  none of the pimcore doc its given about migration installer, not able to find the solution

If anyone come across this error or know about this? please let me know

![installerError](https://github.com/pimcore/pimcore/assets/78895325/8932245c-ea6f-479e-8598-923b237a3702)

**Comment by Thejaswini-Rao-U** [comment]
anyone please suggest me?

in the previous, migration installer  file was there in vendor/, but in the 10.6 version there is no name of file 

which namespace should i use alternative of this ?

this type of error solution, pimcore never documented anywhere? its difficult to find out. 

Pimcore developers should take care of these things

**Comment by dvesh3** [comment, marked as answer]
Hi @Thejaswini-Rao-U `MigrationInstaller` was removed with `Pimcore 10` so you need to switch to `AbstractInstaller` and define the process to execute your bundles migrations based on standard doctrine approach. please search for "migration" in the change list [here](https://pimcore.com/docs/pimcore/10.6/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/index.html#page_Changes)

**Comment by Thejaswini-Rao-U** [comment]
@dvesh3 , Thanks for the response

if i use install & uninstall method which is in AbstractInstaller class in my existing installer to run migrations ?

previous method was used migrateInstall && migrateUninstall in the old migrationInstaller method

**Reply by dvesh3** [reply]
yes, see [changes](https://github.com/pimcore/pimcore/pull/7543/files#diff-7e44b29e837cb8212ed1a9527c0365186cf5f2827d3662d7a5bb145b65565ef5) for more details.

**Reply by Thejaswini-Rao-U** [reply]
Thanks for the response. will use it
